### PR TITLE
Add Announcement for 1.6.0

### PR DESCRIPTION
**Pull request by Tommy-XL**

### CODE REVIEW

File: Patches/AnnouncementPatch.cs [hunk 1]
```diff
@@ -72,7 +73,8 @@ public Announcement ToAnnouncement()
         return result;
     }
     public static List<ModNews> AllModNews = [];
-    public static string ModNewsURL = "https://raw.githubusercontent.com/0xDrMoe/TownofHost-Enhanced/main/Resources/Announcements/modNews-";
+    public static string ModNewsURL = "https://raw.githubusercontent.com/0xDrMoe/TownofHost-Enhanced/AnnouncementFor160/Resources/Announcements/modNews-";
```
Comment (ryuk201198) [inline Patches/AnnouncementPatch.cs:47]: if you change the url, announcement won't be real time, they will have to wait for next release to get the announcement

Comment (Tommy-XL) [reply Patches/AnnouncementPatch.cs:47]: I know, I'm doing this to test

File: Resources/Announcements/modNews-en_US.json [hunk 2]
```diff
@@ -5,8 +5,82 @@
...
+      "Body": ["Placeholder body"],
       "Date": "2024-02-01T02:18:00Z"
+    },
+    {
+      "Number": "100005",
+      "Title": "Town of Host: Enhanced v1.6.0",
+      "Subtitle": "Another big update",
+      "Short": "TOH: Enhanced v1.6.0",
+      "Body": [
+        "<b>【Base】</b>\n - Base on TOH: Enhanced v1.5.1",
```
Comment (ryuk201198) [inline Resources/Announcements/modNews-en_US.json:17]: should it not be 1.5.2?

Comment (Tommy-XL) [reply Resources/Announcements/modNews-en_US.json]: fixed

**Comment by Tommy-XL** [comment]
Now I think everything is ready, all that remains is to synchronize it with Crowdin

**Pull request merged**